spec/system/assessor_interface/reviewing_professional_standing_spec.rb

```diff
@@ -94,7 +94,7 @@ def when_i_click_on_assessment_decision
   end
 
   def when_i_click_on_lops
-    assessor_review_verifications_page.rows.first.link.click
+    find(".govuk-link").click
```

Comment:
Rather than calling `find` we should update the page object (https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/main/spec/support/autoload/page_objects/assessor_interface/review_verifications.rb) with the new structure of the page. We can use the existing page object for tasks lists: https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/main/spec/support/autoload/page_objects/task_list.rb

Here's an example of it being used:

https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/29cbfd24eac87c84b5db1e4f9b363e6edfe9f254/spec/support/autoload/page_objects/teacher_interface/application.rb#L13